frontend/src/components/GovtMasthead/GovtMasthead.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+import { BiChevronDown, BiChevronUp } from 'react-icons/bi'
+import {
+  Box,
+  chakra,
+  Collapse,
+  Flex,
+  Icon,
+  Stack,
+  Text,
+  useBreakpointValue,
+} from '@chakra-ui/react'
+
+import { BxsBank } from '~assets/icons/BxsBank'
+import { BxsLockAlt } from '~assets/icons/BxsLockAlt'
+import Link from '~components/Link'
+
+import { GovtMastheadIcon } from './GovtMastheadIcon'
+import { GovtMastheadItem } from './GovtMastheadItem'
+
+export interface GovtMastheadProps {
+  isOpen: boolean
+  onToggle: () => void
+}
+
+interface GovtMastheadChildrenProps {
+  isMobile: boolean
+  children: React.ReactNode
+  onToggle: () => void
+}
+
+const HeaderBar = ({
+  isMobile,
+  children,
+  onToggle,
+}: GovtMastheadChildrenProps): JSX.Element => {
+  const styleProps = {
+    bg: 'neutral.200',
+    py: { base: '0.5rem', md: '0.375rem' },
+    px: { base: '1.5rem', md: '1.75rem', lg: '2rem' },
+    textStyle: { base: 'legal', md: 'caption-2' },
+    display: 'flex',
+    width: '100%',
+  }
+
+  // Mobile
+  if (isMobile) {
+    return (
+      <chakra.button {...styleProps} onClick={onToggle}>
+        {children}
+      </chakra.button>
+    )
+  }
+
+  // Non-mobile
+  return <Flex {...styleProps}>{children}</Flex>
+}
+
+const HowToIdentify = ({
+  isMobile,
+  children,
+  onToggle,
+}: GovtMastheadChildrenProps): JSX.Element => {
+  // Mobile
+  if (isMobile) {
+    return (
+      <Text color="primary.500" textDecorationLine="underline">
+        How to identify {children}
+      </Text>
+    )
+  }
+
+  // Non-mobile
+  return (
+    <Link
+      tabIndex={0}
+      ariaLabel="Click to expand masthead for more information"
+      onClick={onToggle}
+    >
+      How to identify {children}
+    </Link>
+  )
+}
+
+export const GovtMasthead = ({
+  isOpen,
+  onToggle,
+}: GovtMastheadProps): JSX.Element => {
+  // Custom function for collapsing/expanding the header,
+  // because in mobile you tap the whole header, on desktop only on the link
+  const isMobile =
+    useBreakpointValue({
+      base: true,
+      xs: true,
+      md: false,
+    }) ?? true
+
+  const childrenProps = {
+    isOpen: isOpen,
+    isMobile: isMobile,
+    onToggle: onToggle,
+  }
```

Comment:
all these logic should be the responsbility of `GovtMastheadContainer` and this component should only concern itself with presentation.